catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrHttpWrapper.java

```diff
@@ -74,12 +74,16 @@ private static KeyStore getKeyStore(String location, String password) {
     }
 
     private static String[] getCipherSuites() {
+        String[] cipherSuites = null;
         if (System.getProperty(SecurityConstants.HTTPS_CIPHER_SUITES) != null) {
-            return StringUtils
+            cipherSuites = StringUtils
                     .split(System.getProperty(SecurityConstants.HTTPS_CIPHER_SUITES), ",");
         } else {
-            return SolrServerFactory.DEFAULT_CIPHER_SUITES;
+            if(SolrServerFactory.DEFAULT_CIPHER_SUITES != null) {
+                cipherSuites = SolrServerFactory.DEFAULT_CIPHER_SUITES.toArray(new String[SolrServerFactory.DEFAULT_CIPHER_SUITES.size()]);
+            }
         }
+        return cipherSuites;
```

Comment:
If we keep the null check above, does `SSLConnectionSocketFactory()` support `null` cipher suites? If not, maybe we should log and throw instead?